charts/gardener-extension-validator-gcp/charts/application/templates/validatingwebhook-validator.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
I just noticed that `ValidatingWebhookconfiguration` was promoted to `v1` in K8s 1.16.

Unfortunately, Gardener is tied to a K8s max version of 1.15 (https://github.com/gardener/gardener/issues/1605) which makes this manifest incompatible. Can you please switch to `admissionregistration.k8s.io/v1beta1` here as well as in `example/40-validatingwebhookconfiguration.yaml `?